website/client/app.vue

```diff
@@ -210,6 +211,7 @@ export default {
     BuyModal,
     SelectMembersModal,
     amazonPaymentsModal,
+    Logout,
```

Comment:
we're importing the logout component here but not using it anywhere, can you remove it?